requirements.txt

```diff
@@ -1,4 +1,4 @@
-pytorch-lightning>=1.2.5
+https://github.com/PyTorchLightning/pytorch-lightning/archive/master.zip
```

Comment:
why no specific version ?

Reply:
The changes to make the DataPipeline works are currently not released to pypi or tagged. For now, it is just temporary to get forward and get everything merged. Once done, we will tag a commit on Lightning side and use it in Flash.